tests/x11/multi_users_dm.pm

```diff
@@ -20,15 +20,15 @@ use testapi;
 use utils;
 
 sub ensure_multi_user_target {
-    systemctl 'isolate multi-user.target';
+    type_string "systemctl isolate multi-user.target\n";
```

Comment:
I would have preferred a description here to describe why the command `systemctl` does not work. Otherwise it will not be clear to a person in the future why he should not change this.